packages/esm-morgue-app/src/workspaces/autopsy.scss

```
@@ -0,0 +1,65 @@
+@use '@carbon/layout';
+@use '@carbon/type';
+@use '@carbon/colors';
+@import '~@openmrs/esm-styleguide/src/vars';
+
+.formButton {
+  height: layout.$spacing-09;
+  display: flex;
+  align-content: flex-start;
+  align-items: baseline;
+  min-width: layout.$spacing-09;
+  margin-bottom: layout.$spacing-05;
+}
+
+.formButtonSet {
+  margin-top: layout.$spacing-09;
+  display: flex;
+  justify-content: center;
+}
+
+.formSubHeader {
+  @include type.type-style('heading-02');
+  margin-top: layout.$spacing-05;
+}
+.formContainer {
+  width: 100%;
+}
+.formGrid {
+  margin: 0 layout.$spacing-05;
+  padding: 0rem;
+}
+.form__gender {
```

```suggestion
.formGender {
```

lets use camelCase since this is the standard with OpenMRS 3.x projects

---

Okay sorry, sorted I have removed the workspace since we will be utilizing the form engine as advised